src/test/unit/engine/NetworkFactoryTest.cpp

```diff
@@ -30,15 +30,23 @@
 #include <nupic/engine/Network.hpp>
 #include <nupic/engine/NuPIC.hpp>
 #include <nupic/engine/Region.hpp>
+#include <nupic/os/Path.hpp>
 #include <nupic/utils/Log.hpp>
 
 using namespace nupic;
 
+// macro to ease nesting
+#define GP Path::getParent
+
+const std::string UNIT_TESTS_EXECUTABLE_PATH = Path::getExecutablePath();
+const std::string NUPIC_CORE_PATH =  GP(GP(GP(GP(UNIT_TESTS_EXECUTABLE_PATH))));
+const std::string PATH_TO_FIXTURES = Path::join(NUPIC_CORE_PATH, "/src/test/unit/engine/fixtures");
```

Comment:
Are you sure the second string being joined can start with a slash? This wouldn't work in Python since the forward slash at the start of the second path of the path will throw out the entire first argument:

```
In [2]: os.path.join("/some/start", "/to/a/path")
Out[2]: '/to/a/path'

In [3]: os.path.join("/some/start", "to/a/path")
Out[3]: '/some/start/to/a/path'
```